core/src/main/java/org/opentosca/toscana/core/util/health/TransformerHealthIndicator.java

```diff
@@ -0,0 +1,88 @@
+package org.opentosca.toscana.core.util.health;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.opentosca.toscana.core.csar.Csar;
+import org.opentosca.toscana.core.csar.CsarDao;
+import org.opentosca.toscana.core.plugin.PluginService;
+import org.opentosca.toscana.core.transformation.Transformation;
+import org.opentosca.toscana.core.transformation.TransformationState;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TransformerHealthIndicator extends AbstractHealthIndicator {
+
+    private static final String INSTALLED_PLUGINS_KEY = "installed_plugins";
+    private static final String RUNNING_TRANSFORMATIONS_KEY = "running_transformations";
+    private static final String ERRORED_TRANSFORMATIONS_KEY = "errored_transformations";
```

Comment:
'errored' is not a word. I suggest using: 'failed_transformations'

Reply:
Well the terminology of "failiure" is not the right wording when following these definitions [Click Me](https://vikashazrati.wordpress.com/2008/10/30/fault-failure-error/).

I don't beleve the word "errored" does not exist, because many build tools (e.g. Travis or Gulp) use this wording if the build could not complete because of a exception when building.

Reply:
Today I've seen maven using this terminology. Excuse me, I errored.

Reply:
No problem.